yarn.lock

```diff
@@ -1362,7 +1362,7 @@
     "@types/qs" "*"
     "@types/range-parser" "*"
 
-"@types/express@^4.17.13", "@types/express@^4.17.7", "@types/express@^4.17.8":
+"@types/express@^4.17.13", "@types/express@^4.17.8":
```

Comment:
Along with `package.json`, this diff goes away after rebasing onto the latest `main`.